api/filters/replacement/replacement_test.go

```diff
@@ -1514,9 +1514,9 @@ kind: Deployment
 metadata:
   name: pre-deploy
   annotations: 
-    internal.config.kubernetes.io/previousNames: deploy,deploy
-    internal.config.kubernetes.io/previousKinds: CronJob,Deployment
-    internal.config.kubernetes.io/previousNamespaces: default,default
+    internal.config.kubernetes.io/previousNames: deploy;deploy
```

Comment:
changed the separator for these internal annotations to a colon, because the refBy annotation appends ResIds and I wanted to use commas in the ResId string